packages/web-service/src/pages/eligibility/eligibility.js

```diff
@@ -0,0 +1,172 @@
+/**
+ * Route handlers for the eligibility check flow
+ */
+import { yesNoPage, isYes } from '../common/yes-no.js'
+import { checkAnswersPage } from '../common/check-answers.js'
+import { eligibilityURIs } from '../../uris.js'
+import pageRoute from '../../routes/page-route.js'
+// The pages in the flow
+const {
+  LANDOWNER, LANDOWNER_PERMISSION, CONSENT, CONSENT_GRANTED,
+  NOT_ELIGIBLE_LANDOWNER, NOT_ELIGIBLE_PROJECT, ELIGIBILITY_CHECK, ELIGIBLE
+} = eligibilityURIs
+
+const IS_OWNER_OF_LAND = 'isOwnerOfLand'
+const HAS_LANDOWNER_PERMISSION = 'hasLandOwnerPermission'
+const PERMISSION_REQUIRED = 'permissionsRequired'
+const PERMISSION_GRANTED = 'permissionsGranted'
+
+// Helper to operate on the eligibility section of the journey cache
+export const eligibilityHelper = async (request, operator) => {
```

Comment:
`Helper` isn't clear.

This function seems to get the current eligibility data from the cache, transform it using a provided function and then write it back to the cache.

A more descriptive name should be used - possibly something to articulate that it updates the state?

Reply:
Renamed to updateEligibilityCache